packages/perpetual/index.js

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env node
```

Comment:
2 initial nits:
1) do you want to call the package `Funding-Rate-Proposer` or maybe `Perpetual-Proposer`? IDK, Perpetual does not feel like a good name.
2) do you want to turn this into typescript, seeing this is a new package?

Reply:
1. Renamed to `funding-rate-proposer`
2. Yes, but I wanted to get this out as quickly as possible. I'm thinking of either converting to TS in a subsequent PR or just biting bullet and doing it now. Do you think TS and truffle work well together?

Reply:
On the typescript question, I think I would be in favor of starting with ts if we hadn't already written most of the code. However, since it's already written in js, I would say we should just _convert_ to ts subsequently in the name of unblocking the perp rollout.